### DISCUSSION TITLE
알고리즘 틀린 풀이 커밋에 관하여

### DISCUSSION THREAD

**Discussion by DevingCha** [dongjji/Algorithm#25] [Q&A] [answered]
알고리즘 풀이에 실패한 코드를 올리면 다른사람들이 볼 때 틀린 풀이도 맞는 풀이로 해석할 수 있지 않을까요?

**Comment by devycha** [comment, marked as answer]
알고리즘 틀린 풀이를 올리실 때는 커밋 메시지에 fail을 추가하고 
추가적으로 문제 설명 주석에 틀린 풀이라는 것을 명시해주면 될 것 같습니다

**Reply by DevingCha** [reply]
답변 감사합니다.
